core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/dot-experiments-goals-coming-soon/dot-experiments-goals-coming-soon.component.scss

```diff
@@ -0,0 +1,53 @@
+@use "variables" as *;
+
+:host {
+    display: flex;
+    background: $color-palette-gray-100;
+    flex-direction: column;
+    align-items: center;
+    border-radius: $border-radius-sm;
+    padding: 0.5rem;
```

Comment:
We can use a global spacing variable for this

Reply:
@zJaaal feedback done